### ISSUE TITLE
Brief explanation on adding nodes to the cluster required

### ISSUE THREAD

**Issue by nitishfy** [kubernetes/website#38761]
### Problem:
In the following [docs](https://kubernetes.io/docs/concepts/architecture/nodes/) the information on adding new nodes to the cluster has been mentioned.  However in the [Management](https://kubernetes.io/docs/concepts/architecture/nodes/#management) section, it's  mentioned that:

_There are two main ways to have Nodes added to the [API server](https://kubernetes.io/docs/concepts/overview/components/#kube-apiserver):_

The **Node objects** are added to the API server and not the nodes .  A node object is a representation of a node in the API server.

### Solution:
Update the above docs to briefly mention about Node objects & mention the important line:
_There are two main ways to have node objects added to the API server._

Also, Before this line, mention a little about the node object.

**Comment by NitishKumar06** [comment]
/retitle Brief explanation on adding nodes to the cluster required

**Comment by sftim** [comment]
We could write more text before that paragraph to explain that nodes (servers) are represented by Nodes (objects) in the Kubernetes API.

Would that do it?

**Comment by NitishKumar06** [comment]
Yes this would help a lot. Should I go for it?

**Comment by sftim** [comment]
PRs are welcome!

Could you rephrase the issue description to (briefly) explain the fix you now think is right?

/language en

**Comment by rgunaseknokia** [comment]
@NitishKumar06 Let me know if i can join you for contributing / helping on this issue!

**Comment by sftim** [comment]
Along with this improvement, I suggest that we consider writing a new task page about adding nodes.

See https://github.com/kubernetes/website/issues/37266#issuecomment-1274509225 for context.

**Comment by NitishKumar06** [comment]
> Along with this improvement, I suggest that we consider writing a new task page about adding nodes.
> 
> See [#37266 (comment)](https://github.com/kubernetes/website/issues/37266#issuecomment-1274509225) for context.

@rgunaseknokia Would you like to go with this?

@sftim We can raise a new issue for that or the current one will continue?

**Comment by sftim** [comment]
We can retitle this issue or make a separate issue. Either of those approaches will work.

**Comment by natalisucks** [comment]
@nitishfy Hi there – could you take a look at @divya-mohan0209's [comment](https://github.com/kubernetes/website/issues/38761#issuecomment-1845232863) and, if you agree, modify the issue description so that we can accept it as triaged? Thanks!

**Comment by natalisucks** [comment]
> We can retitle this issue or make a separate issue. Either of those approaches will work.

I agree with @sftim's proposal above, and as we haven't heard back from the issue author for this update to take place per @divya-mohan0209's great feedback, I'll be closing this. For folks interested in addressing the updated need for this change per this comment, please open a new issue!

/close
